goalie/mesh_seq.py

```diff
@@ -68,6 +68,10 @@ def __init__(self, time_partition, initial_meshes, **kwargs):
             self.params = AdaptParameters()
         self.sections = [{} for mesh in self]
 
+        # Set the method for transferring fields between meshes
+        transfer_method = kwargs.get("transfer_method", "interpolate")
+        self.transfer = lambda source, target: transfer(source, target, transfer_method)
```

Comment:
Please write this out explicitly as a method and include a docstring, rather than adding it as a lambda this way.

Reply:
Thanks! Fixed in 78a25163f0e015d33c3c8b466aa4c559e80a7587. I also made it a private method.